app/data/test_navigation.json

```diff
@@ -0,0 +1,667 @@
+{
```

Comment:
I understand we are against the deadline all of the suggestions don't have to be tackled for the changes to go in but things should be raised so they can be picked up later